### DISCUSSION TITLE
Python 表情包

### DISCUSSION THREAD

**Discussion by juliaouo** [1101-GS4719-C/Python-Community-GS4719-C#1502] [優秀Python套件]
可以在terminal輸出emoji，也可以將emoji編碼，在處理含emoji的資料很方便！
#### 安裝
```
pip install emoji
```
#### 使用
```
>>> import emoji
>>> print(emoji.emojize('Python is :thumbs_up:'))
Python is 👍
>>> print(emoji.emojize('Python is :thumbsup:', language='alias'))
Python is 👍
>>> print(emoji.demojize('Python is 👍'))
Python is :thumbs_up:
>>> print(emoji.emojize("Python is fun :red_heart:"))
Python is fun ❤
>>> print(emoji.emojize("Python is fun :red_heart:", variant="emoji_type"))
Python is fun ❤️ #red heart, not black heart
>>> print(emoji.is_emoji("👍"))
True
```
#### 表情包編碼參考
- https://www.webfx.com/tools/emoji-cheat-sheet/
- https://www.unicode.org/emoji/charts/full-emoji-list.html

**Comment by Tzuhsuannnnn** [comment]
好特別!在電腦上我常常只會打XD,:)等等，沒想到也打得出emoji!

**Comment by RogelioKG** [comment]
我記得 Windows 鍵 + . 就可以開啟 emoji，也可以在終端機開啟 emoji 和輸入 emoji。
而且 Python 3 也確實的支援了 Unicode。
那有沒有什麼特別功能是這個模組可以辦到的，
樓主可以多說明一些嗎？

**Comment by lin157644** [comment]
因為在Github的Markdown裡也支援這類用`:emoji:`產生Emoji的方式，有時會在一些如Commit訊息或Release note上看到emoji的使用，我覺得這個套件可能很適合用於自動化這些場景，畢竟ASCII文字處理要比Unicode字元好處理不少。
